test/either.test.js

```diff
@@ -59,6 +59,32 @@ describe('Either', function() {
     jsv.assert(jsv.forall(eNatArb, fnNatArb, fnNatArb, eTest.associative));
   });
 
+  describe('checking for Left | Right', function() {
```

Comment:
Would you mind updating these tests to use the jsverify arbitrary instances for `Either`? You can use the `leftArb` and `rightArb` functions to create `Left` and `Right` instances respectively.

```js
it('should allow the user to check if the instance is a Left', function() {
  jsv.assert(jsv.forall(leftArb(jsv.nat), rightArb(jsv.nat), function(l, r) {
    return l.isLeft() === true && r.isLeft() === false;
  }));
});
```